packages/flutter/lib/src/widgets/scroll_metrics.dart

```diff
@@ -36,7 +36,7 @@ abstract class ScrollMetrics {
   ///
   /// This is useful if this object is mutable, but you want to get a snapshot
   /// of the current state.
-  ScrollMetrics cloneMetrics() => new FixedScrollMetrics.clone(this);
+  ScrollMetrics toMetrics() => new FixedScrollMetrics.clone(this);
```

Comment:
That seems very broad. I'm surprised there aren't more false positives.

I think that's the kind of thing that would be better caught by code review.